src/CMakeLists.txt

```
@@ -35,10 +35,9 @@ endif()
 
 # set compiler flags depending on compiler
 if(MSVC)
-	target_compile_options(${PROJECT_NAME} PRIVATE "/W4" "$<$<CONFIG:RELEASE>:/O2 /GL>")
-	target_link_options(${PROJECT_NAME} PRIVATE "$<$<CONFIG:Release>:/LTCG>")
+	target_compile_options(${PROJECT_NAME} PRIVATE "/W4" "/utf-8" "$<$<CONFIG:RELEASE>:/O2 /GL>")
```

```suggestion
	target_compile_options(${PROJECT_NAME} PRIVATE "/W4" "/utf-8")
```
The `/GL` should be removed as it is linked to IPO and not necessary. It might even be problematic to specify the compile option without specifying the link option.
I would also suggest to remove `/O2` as it is the default level set by CMake for MSVC Release builds.